frontend/src/components/penreg/student/MergeStudents.vue

```diff
@@ -464,8 +505,42 @@ export default {
         return;
       }
 
-      // TODO (jsung) : implement request to student admin api
+      // Status validation
+      if (!this.validateStudentStatuses()) {
+        return;
+      }
+
+      this.isProcessing = true;
```

Comment:
why to use `isProcessing`?

Reply:
working on progress bar....

Reply:
now the flag is integrated with progress bar.